src/Hero.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const Section = styled.section`
+  text-align: center;
+  margin-top: 51px;
+`;
+
+const Title = styled.h1`
+  font-family: 'Bitter', serif;
+  font-weight: normal;
+  font-size: 38px;
+  letter-spacing: 0.8px;
+`;
+
+const Subtitle = styled.p`
+  color: #93918f;
+`;
+
+const Button = styled.button`
+  width: 217px;
```

Comment:
I typically would try not to use a fixed width. Whenever the content of this button changes you'll have to adjust the width as well. Or imagine situations where the font size changes for mobile devices or you support multiple languages resulting in varying text length.

In most situations you should be fine using horizontal padding. It will also make it easier to reuse the component